src/Microsoft.Extensions.ServiceDiscovery.Dns/DnsServiceEndPointResolverBase.Log.cs

```diff
@@ -23,5 +23,8 @@ internal static partial class Log
 
         [LoggerMessage(5, LogLevel.Debug, "DNS SRV query cannot be constructed for service name '{ServiceName}' because no DNS namespace was configured or detected.", EventName = "NoDnsSuffixFound")]
         public static partial void NoDnsSuffixFound(ILogger logger, string serviceName);
+
+        [LoggerMessage(6, LogLevel.Warning, "DNS SRV query returned a record with an unsupported type '{RecordType}' for service name '{ServiceName}'.", EventName = "UnsupportedDnsSrvRecord")]
```

Comment:
Will this always show up in the logs with no way to remove it?

Reply:
Aside from filtering the error out? If this shows up, something is unexpected (eg, bad DNS server implementation). We could throw an exception instead - is that preferable?